crates/blockifier/resources/versioned_constants_0_13_4.json

```diff
@@ -427,6 +431,13 @@
                     "range_check_builtin": 1
                 },
                 "n_memory_holes": 0
+            },
+            "GetClassHashAt": {
```

Comment:
Should I change it to different numbers?

Reply:
May I wait for the os implementation?
<!-- Reviewable comment -OBQ_e3H-azpu2-q96Qu:b-xepoj9 -->
<!-- Sent from Reviewable.io -->

Reply:
Yes
<!-- Reviewable comment -OBZIMTM358zf9w8cDW3:b2nhswy -->
<!-- Sent from Reviewable.io -->